base/array.jl

```diff
@@ -1251,10 +1251,11 @@ julia> findnext(A,3)
 function findnext(A, start::Integer)
     for i = start:length(A)
         if A[i] != 0
-            return i
```

Comment:
I do agree, however, that these cases are clearer with `return`; but they do serve as an in-situ test of the new syntax, which is useful for a WIP pull request.